data/data-pipeline/data_pipeline/etl/score/etl_score_geo.py

```diff
@@ -83,26 +82,33 @@ def extract(self) -> None:
 
     def transform(self) -> None:
         # Rename GEOID10_TRACT to GEOID10 on score to allow merging with Census GeoJSON
-        self.score_usa_df.rename(
-            columns={self.TRACT_SHORT_FIELD: self.GEOID_FIELD_NAME},
-            inplace=True,
-        )
+        # self.score_usa_df.rename(
```

Comment:
temporarily changing this to make it a little faster, happy to revert.

Reply:
Oh I see. This is no longer necessary because you join directly on the indices below. Makes sense to me, I think you can go ahead and delete this line then.